Jenkinsfile_CNP

```diff
@@ -17,4 +18,13 @@ withPipeline(type, product, component) {
   after('test') {
     builder.gradle('integration')
   }
+
+  after('functionalTest:preview') {
```

Comment:
just a single `after('functionalTest')` should do rather than a separate one for preview and aat